### DISCUSSION TITLE
Whole Stack shows up as stopped but it isn't

### DISCUSSION THREAD

**Discussion by lordimac** [louislam/dockge#491] [Ask for Help]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### 📝 Describe your problem

Hello, if one container from the stack is stopped, the whole stack shows up as stopped in dockge.

For example, Zammad is using an init container for updates, preparations, etc. When these tasks are done, the init container is stopped until the next update. dockge shots that the stack is stopped.

Is there any way to ignore the stopped for a container inside the stack?

Thanks!

### 🐻 Dockge Version

1.4.2

### 💻 Operating System and Arch

Ubuntu 22.04

### 🌐 Browser

Firefox 125.0.1

### 🐋 Docker Version

Docker 26.0.1

**Comment by CoMPaTech** [comment]
Would be nice to get some traction on this - most simple way would be ignoring services with `restart: no` I guess?
